drivers/disk/sdmmc_stm32.c

```diff
@@ -718,6 +718,8 @@ static int stm32_sdmmc_pwr_off(struct stm32_sdmmc_priv *priv)
 		return 0;
 	}
 
+	/* PINCTRL sleep mode when powered down */
+	pinctrl_apply_state(priv->pcfg, PINCTRL_STATE_SLEEP);
```

Comment:
Should check the return status.
Maybe `-ENOENT` should be considered as valid, if sleep state is not mandatory.